content/platforms/release-notes/k8s-6-0-8-20-2020-12.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.0.8-20 (December 2020)
+description: Release notes for version 6.0.8-20 (latest)
+weight: 72
+alwaysopen: false
+categories: ["Platforms"]
+---
+
+The Redis Enterprise K8s 6.0.8-20 release is a *major release* on top of [6.0.8-1](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases/tag/v6.0.8-1) providing support for the latest [Redis Enterprise Software release 6.0.8-30](https://docs.redislabs.com/latest/rs/release-notes/rs-6-0-8-september-2020/) and includes several enhancements  and bug fixes.
+
+## Overview
+
+This release of the operator provides:
+
+ * The latest release version of the operator
+ * New features
+ * Various bug fixes
+
+
+Follow the instructions in ["Upgrading a Redis Enterprise Cluster in Operator-based Architecture"]({{< relref "/platforms/kubernetes/upgrading-kubernetes-with-operator.md" >}}) to upgrade to this Kubernetes operator release.
+
+## Images
+
+ * **Redis Enterprise** - redislabs/redis:6.0.8-30 or redislabs/redis:6.0.8-30.rhel7-openshift
+ * **Operator** - redislabs/operator:6.0.8-20
+ * **Services Rigger** - redislabs/k8s-controller:6.0.8-20 or redislabs/services-manager:6.0.8-20 (Red Hat registry)
+
+## New features
+
+ * Databases custom resources (REDB) can be created in separate consumer namespaces from the operator and cluster namespace. The operator deployment is configured to watch specific namespaces for these database REDB CR’s.
+ * The [Gesher admission control proxy](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/admission/GESHER.md) is now certified by Red Hat.
+ * REDB CR’s no longer require a Redis Enterprise cluster name. The name will default to the cluster in the context of the operator.
+ * REC and REDB CR’s are now validated via a schema.
+
+## Important fixes
+
+ * Using database controller (REDB) no longer generates errors:  “failed to update database status" (RED44919)
+ * Issues with configuring replica-of through the database controller (REDB) and TLS have been fixed. (RED48285)
+ * A time out issue with “rlutil upgrade” was fixed. (RED48700)
+
+## Known limitations
+
+### CrashLoopBackOff causes cluster recovery to be incomplete  (RED33713)
+
+When a pod status is CrashLoopBackOff and we run the cluster recovery, the process will not complete. The workaround is to delete the crashing pods manually and recovery process will continue.
+
+### Long cluster names cause routes to be rejected  (RED25871)
+
+A cluster name longer than 20 characters will result in a rejected route configuration as the host part of the domain name exceeds 63 characters. The workaround is to limit cluster name to 20 characters or less.
```

Comment:
```suggestion
A cluster name longer than 20 characters will result in a rejected route configuration because the host part of the domain name will exceed 63 characters. The workaround is to limit cluster name to 20 characters or fewer.
```